### ISSUE TITLE
A few HTMLLint errors

### ISSUE THREAD

**Issue by pdehaan** [mozilla/send#261]
Via text input on https://html5.validator.nu/ (and view-source on https://send.dev.mozaws.net/):

> 1. Error: Bad value localization for attribute `rel` on element `link`: The string localization is not a registered keyword.
>
>    1. From line 13, column 3; to line 13, column 61
>        ```html
>        en-US">↩  <link rel="localization" href="/locales/send.{locale}.ftl">↩  <sc
>        ```
>
> 1. Error: Bad value /locales/send.{locale}.ftl for attribute `href` on element `link`: Illegal character in path segment: { is not allowed.
>
>    1. From line 13, column 3; to line 13, column 61
>        ```html
>        en-US">↩  <link rel="localization" href="/locales/send.{locale}.ftl">↩  <sc
>        ```
>
> 1. Error: An `img` element must have an `alt` attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
>
>     1. From line 18, column 5; to line 18, column 41
>        ```html
>        ogo">↩    <img src="/resources/send_logo.svg"/>↩    <
>        ```
>    1. From line 19, column 5; to line 19, column 46
>        ```html
>        vg"/>↩    <img src="/resources/send_logo_type.svg"/>↩  </d
>        ```
>    1. From line 29, column 26; to line 29, column 87
>        ```html
>        load-img"><img data-l10n-id="uploadSvgAlt" src="/resources/upload.svg"/></div>
>        ```
>    1. From line 91, column 3; to line 91, column 98
>        ```html
>        ></div>↩  <img id="upload-error-img" data-l10n-id="errorAltText" src="/resources/illustration_error.svg"/>↩  <a 
>        ```
>    1. From line 110, column 42; to line 110, column 102
>        ```html
>        illa.org"><img class="mozilla-logo" src="/resources/mozilla-logo.svg"/></a>↩ 
>        ```
>    1. From line 118, column 65; to line 118, column 118
>        ```html
>        ="_blank"><img class="github" src="/resources/github-icon.svg"/></a>↩ 
>        ```
>    1. From line 119, column 65; to line 119, column 120
>        ```html
>        ="_blank"><img class="twitter" src="/resources/twitter-icon.svg"/></a>↩ 
>        ```
>
> 1. Error: The `width` attribute on the `th` element is obsolete. Use CSS instead.
>    1. From line 43, column 46; to line 44, column 57
>         ```html
>        false" -->↩          <th width="35%" data-l10n-id="sentFilesTitle1"></th>↩
>        ```
>    1. From line 44, column 63; to line 45, column 57
>        ```html
>        le1"></th>↩          <th width="25%" data-l10n-id="sentFilesTitle2"></th>↩
>        ```
>    1. From line 45, column 63; to line 46, column 57
>        ```html
>        le2"></th>↩          <th width="21%" data-l10n-id="sentFilesTitle3"></th>↩
>        ```
>    1. From line 46, column 63; to line 47, column 57
>        ```html
>        le3"></th>↩          <th width="12%" data-l10n-id="sentFilesTitle4"></th>↩
>        ```
>
> 1. Error: Bad value `true` for attribute `hidden` on element `div`.
>    1. From line 57, column 1; to line 57, column 40
>        ```html
>        ↩</div>↩↩<div id="upload-progress" hidden="true">↩  <di
>        ```
>    1. From line 75, column 1; to line 75, column 35
>        ```html
>        ↩↩</div>↩↩<div id="share-link" hidden="true">↩  <di
>        ```
>    1. From line 88, column 1; to line 88, column 37
>        ```html
>        >↩</div>↩↩<div id="upload-error" hidden="true">↩  <di
>        ```
>    1. From line 95, column 1; to line 95, column 44
>        ```html
>        >↩</div>↩↩<div id="unsupported-browser" hidden="true">↩  <di
>        ```

**Comment by pdehaan** [comment]
TL;DR:
I think results (1) _"Error: Bad value localization for attribute `rel` on element `<link>`: The string localization is not a registered keyword."_ and (2) _"Error: Bad value `/locales/send.{locale}.ftl` for attribute `href` on element `<link>`: Illegal character in path segment: { is not allowed."_ are bogus.

I don't think (3) _"Error: An `<img>` element must have an `alt` attribute, except under certain conditions."_ is entirely valid either, since I think we're injecting localized `alt=""` attributes through jQuery.

That only leaves (4) _"Error: The `width` attribute on the `<th>` element is obsolete. Use CSS instead."_ and (5) _"Error: Bad value `true` for attribute `hidden` on element `<div>`."_, which I think are semi-valid.

**Comment by uname-yang** [comment]
For `/locales/{{locale}}/send.ftl` , i think we do have load this locale value is `server.js`.  
The `availableLanguages`  in the same html is valid. 

```
app.engine(
  'handlebars',
  exphbs({
    defaultLayout: 'main',
    partialsDir: 'views/partials/',
    helpers: {
      availableLanguages,
      l10nDev: conf.l10n_dev
    }
  })
);
```

**Comment by pdehaan** [comment]
Closing this in favor of #139 meta-bug.
Out of that list above, I believe only (4) and (5) were errors, and they're already being tracked separately as:

- (4) The “width” attribute on the “th” element is obsolete. Use CSS instead #436
- (5) Bad value “true” for attribute “hidden” on element “div” #441
